app/Http/Transformers/SignupTransformer.php

```diff
@@ -26,41 +27,44 @@ class SignupTransformer extends TransformerAbstract
     public function transform(Signup $signup)
     {
         return [
-            'id' => $signup->id,
+            'signup_id' => $signup->id,
+            'signup_event_id' => $signup->event_id,
+            'submission_type' => $signup->event->submission_type,
             'northstar_id' => $signup->northstar_id,
             'campaign_id' => $signup->campaign_id,
             'campaign_run_id' => $signup->campaign_run_id,
             'quantity' => $signup->quantity,
             'quantity_pending' => $signup->quantity_pending,
             'why_participated' => $signup->why_participated,
+            'signup_source' => $signup->source,
             'created_at' => $signup->created_at->toIso8601String(),
             'updated_at' => $signup->updated_at->toIso8601String(),
         ];
     }
 
     /**
-     * Include the event
+     * Include the post
      *
      * @param \Rogue\Models\Signup $signup
-     * @return \League\Fractal\Resource\Item
+     * @return \League\Fractal\Resource\Collection
      */
-    public function includeEvent(Signup $signup)
+    public function includePosts(Signup $signup)
     {
-        $event = $signup->event;
+        $post = $signup->posts;
 
-        return $this->item($event, new EventTransformer);
+        return $this->collection($post, new PostTransformer);
     }
 
     /**
-     * Include the post
+     * Include the event
      *
      * @param \Rogue\Models\Signup $signup
-     * @return \League\Fractal\Resource\Collection
+     * @return \League\Fractal\Resource\Item
      */
-    public function includePost(Signup $signup)
+    public function includeEvents(Signup $signup)
     {
-        $post = $signup->posts;
+        $event = Event::where('signup_id', $signup->id)->get();
```

Comment:
ah I see! Updated both the Signup model and the `SignupTransformer` to reflect this :) @DFurnes @sbsmith86